.github/workflows/gateway-conformance.yml

```
@@ -47,7 +47,6 @@ jobs:
           xml: output.xml
           html: output.html
           markdown: output.md
```

We can stick with example.net :) 
```suggestion
          markdown: output.md
          subdomain-url: http://example.net
```